src/ycc_hull/models/helpers_dtos.py

```diff
@@ -266,6 +264,10 @@ class HelperTaskUpdateRequestDto(HelperTaskMutationRequestBaseDto):
     Update request DTO for helper task.
     """
 
+    notify_signed_up_members: bool = Field(
+        description="Whether to notify signed-up members about the update"
```

Comment:
Probably easier to read if you remove "Whether to"

Reply:
Thanks